email/src/main/java/io/micronaut/email/Attachment.java

```diff
@@ -53,21 +53,41 @@ public class Attachment {
     @Nullable
     private final String id;
 
+    @Nullable
+    private final String disposition;
+
     /**
      *
      * @param filename filename to show up in email
      * @param content file content
      * @param contentType file content type
      * @param id content identifier
+     * @param disposition content disposition
      */
     public Attachment(@NonNull String filename,
```

Comment:
annotate the new constructor wit `@since`. Annotate the primary constructor with `@Creator`